install_config/install/deploy_router.adoc

```diff
@@ -269,9 +280,62 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+An administrator can create multiple routers with the same definition
+that will all serve the same set of routes. By having different groups
+of routers with different namespace or route selectors, they can vary
+the routes that the router will serve.
+
+
+Multiple routers can be grouped to distribute routing load in the cluster and separate
+tenants to different routers.  Each router in the group shards a set of routes that are
```

Comment:
My understanding is that a shard is conceptual construct based on two separate situations: a **route** having a label `foo=bar` and a **router** having a selection expression that specifies `foo=bar`.

I don't think it's accurate to use "shard" as a verb.